docs/INSTALL.K8S.md

```diff
@@ -0,0 +1,41 @@
+Installing Kubernetes
+=====================
+
+On the master node, start etcd
+
+```
+docker run --net=host -d gcr.io/google_containers/etcd:2.0.12 /usr/local/bin/etcd --addr=127.0.0.1:4001 --bind-addr=0.0.0.0:4001 --data-dir=/var/etcd/data
+```
+
+Download the latest stable kubernetes.tar.gz from: https://github.com/kubernetes/kubernetes/releases
+
+Untar the file and look for kubernetes-server-linux-amd64.tar.gz.
+Untar that file too.  Copy kube-apiserver, kube-controller-manager,
+kube-scheduler, kubelet and kubectl to a directory.
+
+On the master node, start the following daemons.
+
+```
+nohup ./kube-apiserver --portal-net=192.168.200.0/24 --address=0.0.0.0 \
```

Comment:
I see that it is now --service-cluster-ip-range. They are maintaining backwards compatibility. I will fix it.